README.md

```diff
@@ -25,14 +53,58 @@ We recommend using Anaconda for developing as follows:
 git submodule update --init --recursive
 
 # Create the environment
-conda create -n autopytorch python=3.8
-conda activate autopytorch
+conda create -n auto-pytorch python=3.8
+conda activate auto-pytorch
 conda install swig
 cat requirements.txt | xargs -n 1 -L 1 pip install
 python setup.py install
```

Comment:
The requirements seem to be read in through `setupy.py`, I'm not sure the step `cat requirements.txt | xargs -n 1 -L 1 pip install` is necessary.

Reply:
yes I think we can remove it.